invenio_administration/assets/semantic-ui/js/invenio_administration/src/search/SearchResultsContainer.js

```diff
@@ -25,8 +25,11 @@ export const SearchResultsContainer = ({
     <Table>
       <Table.Header>
         <Table.Row>
-          {columns.map(([property, { text, order }], index) => {
-            const width = index === 0 ? undefined : index === 1 ? 4 : 3;
+          {columns.map(([property, { text, order, width }], index) => {
+            if (!width) {
+              width = index === 0 ? undefined : index === 1 ? 4 : 3;
+            }
```

Comment:
:+1: 
Maybe in the future we need to find alternative methods to display the grid, as I had to do a custom back-end bound solution in the pages administration panel for it to look nice.